### DISCUSSION TITLE
Where is index.html file located when we start generic site (not a blog)?

### DISCUSSION THREAD

**Discussion by quanta-science** [getnikola/nikola#3575] [Q&A] [answered]
Hi everyone, I am a beginner with Nikola SSG, and don't know some simple things.
I installed virtual environment and Nikola on my hosting, then I started my site (not a blog!)
as per recommendations here (https://getnikola.com/creating-a-site-not-a-blog-with-nikola.html).
After that I could not find index.html file in output folder. The question is "how and/or where to get this file" or "should I make it separately".
Thanks in advance

**Comment by sukiletxe** [comment, marked as answer]
Hi,
If you follow the instructions exactly, when you create the page with `nikola new_page` with the title "index", it should be saved in pages/index.rst and it should be available in output/index.html.

**Reply by quanta-science** [reply]
Thank you very much!
